### PR TITLE
feat :implemented add function

### PR DESCRIPTION
**Pull request by FayazCodeCraft**
- Performs addition of two numbers

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Adds two numbers
```
Comment (ghost) [inline math.js]: Add more description to the function

**Review by ghost — CHANGES_REQUESTED** [review]

File: math.js (hunk 1 above)
Comment (FayazCodeCraft) [reply math.js]: Fixed

**Review by ghost — APPROVED** [review]
LGTM

**Review by vinusuhas-codecraft — APPROVED** [review]
LGTM

**Comment by vinusuhas-codecraft** [comment]
squash your comment

**Pull request merged**